sklearn/linear_model/tests/test_logistic.py

```diff
@@ -443,11 +444,12 @@ def test_ovr_multinomial_iris():
     n_samples, n_features = train.shape
 
     # Use pre-defined fold as folds generated for different y
-    cv = StratifiedKFold(target, 3)
-    clf = LogisticRegressionCV(cv=cv)
+    cv = StratifiedKFold(3)
+    predefined_folds = list(cv.split(train, target))
```

Comment:
From what I can tell, this test conflates classes 0 and 1, and makes sure that what OvR learns for class 2 is the same regardless of whether 0 and 1 are separated or not. I don't see why there's even a need to instantiate a CV class; StratifiedKFold is used by default and it's reproducible.